### DISCUSSION TITLE
RFC: Remove dispatch calls of update commission without chill and remove collator MaxCommission limitation

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1272] [General]
Commission is kind of contract between collator and nominators, most nominators are not educated or informed that the collator can change it without consent of nominators, also no effective ways being notified about the changes (Expecting them checking some app website regularly is not very practical and energy consuming).

More radical solution is not allowing collator update the commission without chill. If he want to change, he need to chill and dismiss original binding, and reinitiate just like a new collator for other nominators to stake with it.

Ref:

https://github.com/darwinia-network/darwinia/blob/31475d9a274b01936186f9092dc8ee0e3c6b2446/pallet/staking/src/lib.rs#L467

https://github.com/darwinia-network/darwinia/pull/1241

https://github.com/paritytech/polkadot-sdk/issues/472

**Comment by Laki389** [comment]
**My thinking is that the maximum commission should remain at 30% and every change of the commission the collator should pay 2 000 Ring which will go to the Treasury.**

**Note:** Since I'm not a developer, I don't know if the idea of the collator paying 2000 Ring (which would go to the Treasury) if he wants to change commission is easily feasible.
(and that in the future it will not cause any technical problems that I cannot predict).

If it is easily achievable, I am in favor of this approach.
**For those who may not know, there are collators who for the duration of the entire Era have 1 or 10% commission, but just when it is necessary they change it to 30%** (because now they cannot go to 90% and so on) **and so the delegates do not know that they don't pay 1% but 30%.**
So from https://app.appsmith.com/app/darwinia/collators-64e6bc932e702a181283cf92  can be seen collators that changed **42** and **27 times** in 7 days.

**Comment by aurexav** [comment]
https://github.com/darwinia-network/DIPs/blob/main/DIPs/dip-1.md
